### ISSUE TITLE
Specify Sqlite datatype

### ISSUE THREAD

**Issue by porfirioribeiro** [greenbits/brick#42]
As we have `fromGenerator` and `toGenerator`  to completely change the was we store and retrieve from the database would be nice to be able to specify the column type in the database

Like: 
`Sqlite(type: Column.text)`

This would give more flexibility and let do some advanced stuff

**Comment by tshedor** [comment]
@porfirioribeiro good news, and hopefully not too late, I think I've got a functional solution to this in #44 . Would you mind giving it a trial for me? 

You'll need to do a little dependency magic to use this branch in your `pubspec.yaml`:

```yaml
dependency_overrides:
  brick_sqlite_abstract:
    git:
      url: git@github.com:greenbits/brick.git
      path: packages/brick_sqlite_abstract
      ref: add-uint8list-support
  brick_sqlite_generators:
    git:
      url: git@github.com:greenbits/brick.git
      path: packages/brick_sqlite_generators
      ref: add-uint8list-support
```

**Comment by porfirioribeiro** [comment]
That's good news! I'm not currently working on that part of the app, but I'll try to take some time to test it when I can!

Thanks!

**Comment by tshedor** [comment]
Resolved in #44
